api-call.py

```diff
@@ -3,6 +3,27 @@
 
 
 
+
+def get_access_token():
+    url = "https://api.oregonstate.edu/oauth2/token"
+    payload = {
+        "grant_type": "client_credentials",
+        "client_id": "8HAveOfxhbEQBDD5sA0VCEKVs8AHBv6z",
+        "client_secret": "tcGxL8lztWYQYjbm"
```

Comment:
You should be very careful of what you push to git. For security concerns, any sensitive data shouldn't be hardcoded and pushed to git history. Instead, you should put these data into a configuration file and then load these data from the config file. Make sure the config file is also [gitignored](https://git-scm.com/docs/gitignore).

Question to think: what else you should put in a config file and how can you improve your code?

Reply:
this is updated, please let me know if there's anything else that should be in config. I moved whatever fits right for config and based on articles on google.